src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -231,6 +231,7 @@ public partial class COMException : System.Exception
         public COMException(string message) { }
         public COMException(string message, System.Exception inner) { }
         public COMException(string message, int errorCode) { }
+        public override string ToString() { return default(string); }
```

Comment:
As long as you're adding this, can you add the ```protected COMException(SerializationInfo info, StreamingContext context)``` ctor as well and change the type to derive from ExternalException?